.jshintignore

```diff
@@ -58,6 +58,7 @@ tv_apps/notification-sender/bower_components/**
 tv_apps/smart-home/bower_components/**
 tv_apps/smart-settings/bower_components/**
 tv_apps/smart-system/bower_components/**
+tv_apps/tv-deck/bower_components/**
```

Comment:
Please turn on the jshint of new app. We should follow the jshint  while writing new modules or apps.